Makefile

```diff
@@ -7,16 +7,21 @@
 RUN= # used by go tests to decide which tests to run (i.e. passed to -run)
 # Label it w the go version we bundle in:
 export VERSION_ADDITIONAL = -$(shell git log --pretty=format:%H | head -n 1)
-LD_FLAGS = -X github.com/pachyderm/pachyderm/src/client/version.AdditionalVersion=$(VERSION_ADDITIONAL)
+export LD_FLAGS = github.com/pachyderm/pachyderm/src/client/version.AdditionalVersion=$(VERSION_ADDITIONAL)
 export GC_FLAGS = "all=-trimpath=${PWD}"
 export DOCKER_BUILD_FLAGS
 
-CLUSTER_NAME?=pachyderm
-CLUSTER_MACHINE_TYPE?=n1-standard-4
-CLUSTER_SIZE?=4
+CLUSTER_NAME ?= pachyderm
+CLUSTER_MACHINE_TYPE ?= n1-standard-4
+CLUSTER_SIZE ?= 4
 
-MINIKUBE_MEM=8192 # MB of memory allocated to minikube
-MINIKUBE_CPU=4 # Number of CPUs allocated to minikube
+MINIKUBE_MEM = 8192 # MB of memory allocated to minikube
+MINIKUBE_CPU = 4 # Number of CPUs allocated to minikube
+
+CHLOGFILE = /tmp/pachyderm/release/changelog.diff
+export GOVERSION = $(shell cat etc/compile/GO_VERSION)
+GORELSNAP = #--snapshot # uncomment --snapshot if you want to do a dry run.
+SKIP = #\# # To skip push to docker and github remove # in front of \#
```

Comment:
**Note for future**:This also would probably go into its own PR anyway, but as a matter of ergonomics, do you think it would be controlling both of these with a single var? e.g.
```
ifdef DRY_RUN
  GORELEASER_FLAGS += --snapshot
  DOCKER_SKIP = \#
endif
```